book/src/keymap.md

```diff
@@ -166,6 +166,9 @@ These sub-modes are accessible from normal mode and typically switch back to nor
 | `Ctrl-w` | Enter [window mode](#window-mode)                  | N/A            |
 | `Space`  | Enter [space mode](#space-mode)                    | N/A            |
 
+These modes (except `v` and `:`) can be configured by
```

Comment:
Command mode `:` can't currently be remapped but you can remap select mode - both the bindings in select mode:

```toml
[keys.select]
w = "extend_next_long_word_start"
```

and the binding to switch into select-mode itself:

```toml
[keys.normal]
esc = "select_mode"
```

Reply:
right